tests/xtimer_now_irq/tests/01-run.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2019 Freie Universität Berlin,
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+# @author      Julian Holzwarth <julian.holzwarth@fu-berlin.de>
+
+import sys
+import pexpect
+from testrunner import run
+
+
+def testfunc(child):
+    child.expect("OK", timeout=75*60)
```

Comment:
The first 4 expects could be called in a for loop and replaced by `expect_exact` since no regexp is used:
```python
for _ in range(4):
    child.expect_exact("OK", timeout=TIMEOUT)
```

Define TIMEOUT outside of the `testfunc` function. By the way, how do determine the timeout value ?

Reply:
The number is selected to roughly cover a 32 bit timer running at 1 MHz (~72 minutes), right?

Reply:
yes @MichelRottleuthner